src/commands.cc

```diff
@@ -2291,6 +2292,8 @@ const CommandDesc menu_cmd = {
                 select_cmds.push_back(parser[i+2]);
         }
 
+        auto on_abort = parser.get_switch("on-abort").value_or("").str();
```

Comment:
It makes sense to support this, `prompt` has the same switch.

it doesn't feel great to use the empty string as sentinel value, it would be better to use the empty optional